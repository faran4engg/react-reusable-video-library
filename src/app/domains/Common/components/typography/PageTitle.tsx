import { FC } from 'react';
import { PageTitleProps } from './types';

const PageTitle: FC<PageTitleProps> = ({ children }) => (
  <h1 className="my-3 font-semibold text-gray-500 text-1xl dark:text-gray-300">
    {children}
  </h1>
);

export default PageTitle;
