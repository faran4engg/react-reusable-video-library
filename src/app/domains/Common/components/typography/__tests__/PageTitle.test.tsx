import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import PageTitle from '../PageTitle';

const renderer = createRenderer();

describe('<PageTitle />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<PageTitle>This is Page Title</PageTitle>);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
