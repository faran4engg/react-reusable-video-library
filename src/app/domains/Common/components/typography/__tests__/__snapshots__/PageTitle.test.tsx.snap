// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> should render and match the snapshot 1`] = `
<h1
  className="my-3 font-semibold text-gray-500 text-1xl dark:text-gray-300"
>
  This is Page Title
</h1>
`;
